tracer.go

```diff
@@ -290,11 +291,13 @@ func (t *tracer) startSpanInternal(
 	tags opentracing.Tags,
 	newTrace bool,
 	rpcServer bool,
+	references []opentracing.SpanReference,
```

Comment:
sorry, that's not what I meant. I actually liked your previous change in how it handled the other ref types. We want to make sure that parent_id is set from the first child-of reference, or from follows-from reference if no child-of available.

and for internal array I was suggesting our own type
```
type Reference strict {
    Type opentracing.RefType
    Context jaeger.SpanContext // since we already made this check
}
```